prime-router/src/test/kotlin/fhirengine/azure/FHIRConverterIntegrationTests.kt

```diff
@@ -454,15 +454,15 @@ class FHIRConverterIntegrationTests {
             assertThat(reportAndBundles).transform { pairs -> pairs.map { it.second } }.each {
                 it.matchesPredicate { bytes ->
                     val invalidHL7Result = CompareData().compare(
-                        bytes.inputStream(),
```

Comment:
Only changed these because "expected" is first in `CompareData().compare(...)` and the error it was giving me was throwing me off because of that.